.github/workflows/pr-lint-test-js.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint and tests for JS packages and woocommerce-admin/client
+
+on:
+  pull_request:
+    paths:
+      - 'packages/js/**/**'
+      - 'plugins/woocommerce-admin/client/**'
+      - '!**.md'
+
+jobs:
+  lint-test-js:
+    runs-on: ubuntu-18.04
```

Comment:
Maybe add a `name: Lint and Test JS` so that it doesn't show as `lint-test-js` within Github

Reply:
Added in 71ba4c748a8267e34e9589c809b1bc7126a6fd00.